anyway/infographics_utils.py

```diff
@@ -776,6 +776,16 @@ def __init__(self, request_params: RequestParams):
     def generate_items(self) -> None:
         self.items = ["vision_zero_2_plus_1"]
 
+@register
+class Road2Plus1Widget(Widget):
+    name: str = "vision_zero_2_plus_1"
+
+    def __init__(self, request_params: RequestParams):
+        super().__init__(request_params, type(self).name)
+        self.rank = 24
+
+    def generate_items(self) -> None:
+        self.items = ["vision_zero_2_plus_1"]
```

Comment:
Two widgets with same content?
What do you mean by "different Widget for each vision zero widget"?

Reply:
I understood that the old one will be deleted - #1693